src/main/webapp/app/entities/drag-item/drag-item-dialog.html

```diff
@@ -25,6 +25,11 @@ <h4 class="modal-title" id="myDragItemLabel" data-translate="artemisApp.dragItem
                     ng-model="vm.dragItem.text"
                     />
         </div>
+        <div class="form-group">
+            <label class="control-label" data-translate="arTeMiSApp.dragItem.invalid" for="field_invalid">Invalid</label>
```

Comment:
please make sure to use the right spelling for "artemisApp" (consistent to all other occurrences in the JS code)